test/commands/app/delete/action.test.js

```diff
@@ -13,81 +13,319 @@ const fs = require('fs-extra')
 
 const TheCommand = require('../../../../src/commands/app/delete/action')
 const BaseCommand = require('../../../../src/BaseCommand')
+const cloneDeep = require('lodash.clonedeep')
+const path = require('path')
 
-jest.mock('fs-extra')
+// application.runtimeManifest.packages.pkga.actions.0
+const mockConfigData = {
```

Comment:
nitpick2 that's fine but you could have also used the mocks available in tests/data-mocks/config-loader